### PR TITLE
Add the dialect "Esse ome" on Potiguares

### PR DESCRIPTION
**Pull request by MoisesAbraao**
## Pre-Submission Checklist

- You have only one commit (if not, squash them into one commit).
- All new and existing tests pass the command `yarn test` and `yarn lint`.

### CODE REVIEW

File: dialects/potiguares.json [hunk 1]
```diff
@@ -359,5 +359,16 @@
          "Tive que andar até a baixa da égua hoje atrás de uma pizzaria aberta.",
          "Boy, bati lá na baixa da égua hoje pra participar de um congresso."
       ]
-   }
+   },
+   {
```
Comment (mvfsillva) [inline dialects/potiguares.json]: The tests and linter are ok?

Comment (MoisesAbraao) [reply dialects/potiguares.json]: Yes, it's okay.

**Comment by mvfsillva** [comment]
@MoisesAbraao resolve the conflicting files, please.

**Comment by mvfsillva** [comment]
> So, I fixed the indentation.

Still in conflict: /

**Comment by mvfsillva** [comment]
@MoisesAbraao do you still intend to resolve the conflict?

**Pull request closed without merging**